### PR TITLE
manual: minor improvements

### PR DESCRIPTION
**Pull request by Araq**

### CODE REVIEW

File: doc/manual.rst [hunk 1]
```diff
@@ -482,19 +482,21 @@ literals:
...
-programs will still handle UTF-8 properly as UTF-8 was specially designed for
-this. Another reason is that Nim can thus support `array[char, int]` or
-`set[char]` efficiently as many algorithms rely on this feature.  The `Rune`
-type is used for Unicode characters, it can represent any Unicode character.
+A character is not a Unicode character but a single byte.
+
+Rationale: It enables the efficient support of `array[char, int]` or
+`set[char]`.
+
+The `Rune` can represent any Unicode character.
```
Comment (konsumlamm) [inline doc/manual.rst]:
```suggestion
The `Rune` type can represent any Unicode character.
```

File: doc/manual.rst [hunk 2]
```diff
@@ -1013,7 +1015,9 @@ intXX
   Literals of these types have the suffix 'iXX.
 
 `uint`
-  the generic `unsigned integer`:idx: type; its size is platform-dependent and has the same size as a pointer. An integer literal with the type suffix `'u` is of this type.
+  the generic `unsigned integer`:idx: type; its size is platform-dependent and
+  has the same size as a pointer. An integer literal with the type
```
Comment (konsumlamm) [inline doc/manual.rst:1019]:
```suggestion
  the same as the size of a pointer. An integer literal with the type
```

Comment (Araq) [reply doc/manual.rst:1019]: I think your correction is worse.

Comment (konsumlamm) [reply doc/manual.rst:1019]: "its size is platform-dependent and has the same size as a pointer" means "its size has the same size as a pointer", which doesn't make sense. An alternate formulation would be
```suggestion
  it has the same size as a pointer. An integer literal with the type
```

**Pull request merged**